install_config/master_node_configuration.adoc

```diff
@@ -931,11 +931,13 @@ to limit filesystem permissions and access to the key file.
 [[creating-new-configuration-files]]
 
 == Creating New Configuration Files
-For masters, the `openshift start` command accepts options that indicate that it
-should simply write the configuration files that it would have used, then
-terminate. For nodes, a configuration file can be written using the `oadm
-create-node-config` command. Creating new configuration files is useful to get a
-starting point for defining your configuration.
+
+Creating new configuration files is useful to get a starting point for defining
```

Comment:
This sentence is a little awkward to me. Maybe we can edit it to something like the following? 
To define your configuration, start by creating new configuration files.